src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -199,24 +199,25 @@
       <OtherRunnerScriptArgs>--perf-runner Microsoft.DotNet.xunit.performance.runner.Windows $(OtherRunnerScriptArgs)</OtherRunnerScriptArgs>
     </PropertyGroup>
     <!-- now gather the perf tests -->
+    <!-- All built tests are under bin/tests dir -->
     <ItemGroup>
-      <TestBinary Include="$(BinDir)$(OSPlatformConfig)/**/*.dll" />
-      <TestBinary Include="$(BinDir)$(OSPlatformConfig)/**/*.exe" />
-      <TestBinary Include="$(BinDir)$(AnyOSPlatformConfig)/**/*.dll" />
-      <TestBinary Include="$(BinDir)$(AnyOSPlatformConfig)/**/*.exe" />
+      <TestBinary Include="$(BinDir)tests/$(OSPlatformConfig)/**/*.dll" />
+      <TestBinary Include="$(BinDir)tests/$(OSPlatformConfig)/**/*.exe" />
+      <TestBinary Include="$(BinDir)tests/$(AnyOSPlatformConfig)/**/*.dll" />
+      <TestBinary Include="$(BinDir)tests/$(AnyOSPlatformConfig)/**/*.exe" />
     </ItemGroup>
     <ItemGroup Condition="'$(TargetsUnix)' == 'true'" >
-      <TestBinary Include="$(BinDir)$(UnixPlatformConfig)/**/*.dll" />
-      <TestBinary Include="$(BinDir)$(UnixPlatformConfig)/**/*.exe" />
+      <TestBinary Include="$(BinDir)tests/$(UnixPlatformConfig)/**/*.dll" />
+      <TestBinary Include="$(BinDir)tests/$(UnixPlatformConfig)/**/*.exe" />
     </ItemGroup>
     <GetPerfTestAssemblies TestBinaries="@(TestBinary)" GetFullPaths="false">
       <Output TaskParameter="PerfTestAssemblies" ItemName="PerfTestAssembly" />
     </GetPerfTestAssemblies>
     <!-- don't add any items to the group if no perf tests were found -->
     <ItemGroup Condition="'@(PerfTestAssembly->Count())' != '0'">
-      <PerfTest Include="$(TestArchivesRoot)%(PerfTestAssembly.Identity).zip" />
-      <PerfTest Include="$(AnyOSTestArchivesRoot)%(PerfTestAssembly.Identity).zip" />
-      <PerfTest Condition="'$(TargetsUnix)' == 'true'" Include="$(UnixTestArchivesRoot)%(PerfTestAssembly.Identity).zip" />
+      <PerfTest Include="$(TestArchivesRoot)%(PerfTestAssembly.Identity).zip" Condition="Exists('$(TestArchivesRoot)%(PerfTestAssembly.Identity).zip')"/>
```

Comment:
The existence of the Exists() check here indicates we might not be constructing the items correctly earlier as you're explicitly forming the list just a few lines above.  I get why though... you're checking for the file in every location every time.    Why not just use FullPath instead of Identity (see https://msdn.microsoft.com/en-us/library/ms164313.aspx?f=255&MSPPError=-2147217396 ) and not have the conditional check?